AnnotationPreprocessing/nextflow.config

```diff
@@ -10,6 +10,21 @@ manifest {
 
 profiles {
 
+    uppmax {
+        executor {
+            name = 'slurm'
+        }
+        process {
+            scratch = '$SNIC_TMP'
+        }
+        includeConfig "$baseDir/config/compute_resources.config"
+        env.PERL5LIB='${PERL5LIB:-""}:${HOME}/git/NBIS/GAAS/annotation/'
```

Comment:
```
      includeConfig "$baseDir/config/compute_resources.config"
         env.PERL5LIB='${PERL5LIB:-""}:${HOME}/git/NBIS/GAAS/annotation/'
```
As we are using GAAS from conda It should be removed, isn't it?

Reply:
Good point, yes.